### PR TITLE
Add tokenizer function

### PR DESCRIPTION
**Pull request by darkHarry**
Adding the tokenizer function.

### CODE REVIEW

File: src/state.rs [hunk 1]
```diff
@@ -65,4 +65,8 @@ impl Parser {
         parser.nextToken();
         parser.parseExpression()
     }
+
+    pub fn tokenizer(input: String, options: Option<option::Options>) -> node::Node {
+        Parser::new(options.unwrap_or_default(), input)
+    }
```
Comment (ryzokuken) [inline src/state.rs:71]: That's... it?

Comment (darkHarry) [reply src/state.rs:71]: https://github.com/acornjs/acorn/blob/master/acorn/src/state.js#L121

Comment (ryzokuken) [reply src/state.rs:71]: wait, how does `Parser::new` return `node::Node`?

Comment (darkHarry) [reply src/state.rs:71]: How is your `parse` and `parseExpressionAt` functions returning `node::Node` ?

**Comment by ryzokuken** [comment]
Closing in favor of https://github.com/ryzokuken/cap/pull/6

**Pull request closed without merging**